db/change_cache.go

```diff
@@ -306,25 +318,25 @@ func (c *changeCache) DocChanged(event sgbucket.FeedEvent) {
 	timeReceived := channels.NewFeedTimestamp(&event.TimeReceived)
 	// ** This method does not directly access any state of c, so it doesn't lock.
 	// Is this a user/role doc for this database?
-	if strings.HasPrefix(docID, c.metaKeys.UserKeyPrefix()) {
+	if docType == DocTypeUser {
```

Comment:
Switch/case is going to be easier to read here (and probably slightly more performant) now that there's `DocumentType` to switch on